src/applications/ivc-champva/10-7959C/tests/10-7959C.cypress.spec.js

```diff
@@ -85,7 +85,7 @@ const testConfig = createTestConfig(
             cy.get('va-text-input')
               .shadow()
               .get('#inputField')
-              .type(data.signature);
+              .type(data.signature, { force: true });
```

Comment:
Fixes flaky issue where the signature text field was falsely flagged as `disabled`. ([See this GH thread](https://github.com/cypress-io/cypress/issues/5830))